Formula/ponysay.rb

```diff
@@ -3,7 +3,8 @@ class Ponysay < Formula
   homepage "https://github.com/erkin/ponysay/"
   url "https://github.com/erkin/ponysay/archive/3.0.3.tar.gz"
   sha256 "c382d7f299fa63667d1a4469e1ffbf10b6813dcd29e861de6be55e56dc52b28a"
-  revision 4
+  revision 5
+  head "https://github.com/erkin/ponysay.git"
```

Comment:
Could you please put all `HEAD` related stuff in a `head do` block?

Example: https://github.com/Homebrew/homebrew-core/blob/6a39dfa5a1b2c379f41b2a2f55c8baae75e0d553/Formula/elinks.rb#L15

Reply:
I just checked and we didn't build a bottle for this formula on Linux. Doing it now, here:

https://github.com/Homebrew/linuxbrew-core/pull/20470

If those new dependencies are really needed, then okay, let's leave it as is. Just discard my previous suggestion about `head do` block.